### ISSUE TITLE
Field input error is not announced in full - Siteimprove - CKEditor live region

### ISSUE THREAD

**Issue by joewhitsitt** [uiowa/uiowa#9506]
From #9462

Field input error is not announced in full
3.3.1 Error Identification A
4.1.3 Status Messages AA

CKEditor live announce region have an aria-live assertive value. There seems to be two of these per page. We changed this to polite for uiowa_hours recently but since CKEditor also provides polite containers, I don' think that is the solution here. https://github.com/uiowa/uiowa/pull/9426/changes

<img width="1270" height="842" alt="Image" src="https://github.com/user-attachments/assets/a53e106d-b5e8-43d8-9fca-874bea887593" />

Unless, adding aria-atomic=true to this assertive divs makes SiteImprove happy, I think this is another situation where SiteImprove doesn't like the look of this but would be up to CKEditor to adjust.
